_config.yml

```diff
@@ -6,7 +6,7 @@ google_analytics: UA-125947352-1
 gems:
    - jekyll-paginate
 
-latestGafferVersion: 0.57.0.0
+latestGafferVersion: 0.57.5.0
 copyrightYear: 2019
```

Comment:
should we bump this to 2020 while we're at it?

Reply:
> should we bump this to 2020 while we're at it?

Good idea, thanks. Done.